### ISSUE TITLE
UA.on('connecting') always reports 'attempts = 1'

### ISSUE THREAD

**Issue by ibc** [versatica/JsSIP#313]
Just restart the WebSocket N times (in server side). JsSIP logs are always:

```
JsSIP:Transport trying to reconnect to WebSocket xxxxxxxxx (reconnection attempt 1)
```

and `data.attempts` in the UA's `connecting` event is also always 1.

**Comment by jmillan** [comment]
To be closed as soon as #374 is merged

**Comment by jmillan** [comment]
closing..
